### DISCUSSION TITLE
error ketika read data

### DISCUSSION THREAD

**Discussion by Davidlimbong** [teamalgoritma/community#5] [Q&A] [answered]
saya error ketika read data, mohon bantuannya

```{r}
read.csv("data/q.csv")
```

errornya 

```
Error in file(file, "rt") : cannot open the connection
```

**Comment by NabiilahArdini** [comment, marked as answer]
Halo Bapak David, 

spertinya error karena file tidak berada di working directory yang tepat. pastikan file pathnya betul ketika read data.
